### PR TITLE
Add initial source code

### PR DESCRIPTION
**Pull request by TeknoVenus**
Add the initial BundleGen source code.

Signed-off-by: Stephen Foulds <stephen.foulds@consult.red>

### CODE REVIEW

**Comment by mhughesacn** [comment]
Hi Stephen,
Requesting two small changes please:
1. Rename CONTRIBUTING to CONTRIBUTING.md
2. Create a soft link called COPYING to LICENSE at top level.
Apart from that, this is good - thank you.

Martin
Martin Hughes RDK CMF Compliance Team

File: docs/Templates.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* `logging`
+  * `mode` (string, REQUIRED). Sets the logging sink for containers. Dobby by default supports the following logging modes:
+    * `file` - log to a file
+    * `journald` - log to journald
+    * `devnull` - log to /dev/null (i.e. don't store container logs)
+  * `logDir` (string, REQUIRED if `mode` is set to `file`). Directory to store container logs if logging to file
+* `dobby`
+  * `pluginDir` (string, REQUIRED). Location where Dobby plugins are found on the platform. Dobby plugins are by default found in `/usr/lib/plugins/dobby`
+  * `pluginDependencies` (array of strings, REQUIRED. Libraries that the Dobby plugins depend on.
+    * Dobby plugins are shared objects and have dependencies that must exist in the container. THe exact dependencies needed will depend on which plugins are being used on the platform. To find dependencies, run `/lib/ld-linux-armhf.so.3 --list /usr/lib/plugins/dobby/ <plugin-name>` on the platform for each plugin
```
Comment (anttikyl) [inline docs/Templates.md:66]: Typo (THe)

**Review by anttikyl — COMMENTED** [review]
Aside from a couple of typos, LGTM. Documentation looks great!

**Review by anttikyl — APPROVED** [review]
LGTM

**Pull request merged**